### ISSUE TITLE
[BUG] <A typographical error on docs/contributing/commits page >

### ISSUE THREAD

**Issue by devmamunur** [EddieHubCommunity/BioDrop#9525]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

https://www.biodrop.io/docs/contributing/commits

### Version of BioDrop (for example "v1.2.3")

v2.75.1

### Description

I found a typographical error on this page: https://www.biodrop.io/docs/contributing/commits

1) Continious  will be Continuous

### Screenshots

![Screenshot_1](https://github.com/EddieHubCommunity/BioDrop/assets/49125977/f0555aba-f2ef-415c-95d8-a2b0b57fb0b0)

### Do you want to work on this issue?

Yes

**Comment by SaraJaoude** [comment]
Thank you for raising. I understand you would like to work on this but as it is a good first issue (and you have recently fixed a typo as well), we will be assigning this to someone who is new to the project.

**Comment by Giuliano1993** [comment]
Can I have this assigned? :)

**Comment by trba1810** [comment]
Can assign it to me?

**Comment by Khalid-212** [comment]
hey @devmamunur can i work on this issue

**Comment by rohitmondal03** [comment]
@SaraJaoude can I work on this ?

**Comment by Koushik1311** [comment]
I would like to work on this. can you assign this to me?

**Comment by CodeZera11** [comment]
@SaraJaoude I recently found this repo. I just wanted to ask, do I have to wait for issues to be assigned, or can I raise a PR if no one else has done so?

**Comment by Anushka-Bhowmick** [comment]
Hello, @SaraJaoude I have been eagerly waiting to contribute something to biodrop. Can I please be assigned this issue?

**Comment by Divjot02** [comment]
Hello @SaraJaoude , I am excited to contribute something to biodrop and I'd like to request that the issue be assigned to me as I'm new to the project and eager to get involved.

**Comment by SaraJaoude** [comment]
> SaraJaoude I recently found this repo. I just wanted to ask, do I have to wait for issues to be assigned, or can I raise a PR if no one else has done so?

@CodeZera11 yes you do.  I would suggest that you familiarise yourself with the project's [Contributing Guide](https://github.com/EddieHubCommunity/BioDrop/blob/main/CONTRIBUTING.md) which will explain all you need to know.

**Comment by SaraJaoude** [comment]
> Can you assign it to me?

@trba1810 I will assign this to you
